src/Aspire.Hosting.Azure/AzureEventHubsResource.cs

```diff
@@ -17,13 +17,13 @@ public class AzureEventHubsResource(string name, Action<ResourceModuleConstruct>
     internal List<(string Name, Action<IResourceBuilder<AzureEventHubsResource>, ResourceModuleConstruct, EventHub>? Configure)> Hubs { get; } = [];
 
     /// <summary>
-    /// Gets the "serviceBusEndpoint" output reference from the bicep template for the Azure Event Hubs resource.
+    /// Gets the "eventHubsEndpoint" output reference from the bicep template for the Azure Event Hubs resource.
     /// </summary>
-    public BicepOutputReference ServiceBusEndpoint => new("serviceBusEndpoint", this);
+    public BicepOutputReference EventHubsEndpoint => new("eventHubsEndpoint", this);
```

Comment:
👍🏾

Reply:
I'm going to break the mold here and keep it as eventHubsEndpoint unless otherwise advised. The experienced people will realize under the covers it's actually serviceBusEndpoint, and the newbies will just be less confused (until they find out the former lol)